isotovideo

```diff
@@ -438,6 +441,23 @@ sub process_command {
         myjsonrpc::send_json($bmwqemu::backend->{to_child}, {cmd => $cmd, arguments => $rsp});
         return;
     }
+    if ($rsp->{cmd} eq 'report_timeout') {
+        if (!$pause_on_assert_screen_timeout) {
+            myjsonrpc::send_json($io_handle, {ret => 0});
+            return;
+        }
+
+        $test_execution_paused = $rsp->{msg};
+
+        # emit info
+        myjsonrpc::send_json($cmd_srv_fd, {paused => $rsp, reason => $test_execution_paused});
+        print("isotovideo: pausing test execution at $current_test_full_name because a timeout occurred\n");
```

Comment:
You should  use the diag function instead of print whenever possible, so that it ends on the logs properly on the other hand: 
i'd write the message as: 'pausing test execution on timeout as requested at $current_test_full_name'

Reply:
Ok, I'll change that.